packages/server/graphql/public/mutations/updateSAML.ts

```diff
@@ -0,0 +1,36 @@
+import getRethink from '../../../database/rethinkDriver'
+import {MutationResolvers} from '../resolverTypes'
+import getSignOnURL from './helpers/SAMLHelpers/getSignOnURL'
+
+const updateSAML: MutationResolvers['updateSAML'] = async (_source, {orgId, metadata}, {}) => {
```

Comment:
@Dschoordsch just making sure as I don't see anything subscriptions related, is this mutation supposed to publish any data on `org` level? Or is the UI only for the `billing leader` and there's no need to do anything subscriptions related?

Reply:
The UI is only for the billing leader. There could be multiple, but that's an edge case we don't need to fix right now.

Reply:
got it